tsa/pdf_to_json/main.py

```diff
@@ -17,8 +63,9 @@ def extract_json_from_pdf(bucket_name, pdf_date):
     for i, pdf_date_blob in enumerate(pdf_date_blobs, start=1):
```

Comment:
if you skip the first you could also pass in `pdf_date_blobs[1:]`?

Reply:
good call